vpd-tool/src/vpd_tool.cpp

```diff
@@ -85,4 +86,47 @@ int VpdTool::readKeyword(const std::string& i_vpdPath,
     }
     return l_rc;
 }
+
+int VpdTool::writeKeyword(const std::string& i_vpdPath,
+                          const std::string& i_recordName,
+                          const std::string& i_keywordName,
+                          const std::string& i_keywordValue,
+                          const bool i_onHardware,
+                          const std::string& i_filePath)
+{
+    int l_rc = constants::FAILURE;
+    try
+    {
+        if (i_vpdPath.empty() || i_recordName.empty() || i_keywordName.empty())
```

Comment:
Why are we not checking for `i_keywordValue` empty? Is it a valid input?

Reply:
keyword Value can be read from i_keywordValue or from i_filePath.

Even if I add this check at line no 100 for empty and returning there, will get removed in the next PR, where read value from file if exists.
If couldn't get key value either from i_keywordValue or from reading i_filePath file I will get notified in the next PR.

Reply:
I am not sure what's happening in another PR. From this PR perspective, if the value is empty what are you going to write?